.github/workflows/ci-vulnerability-scans.yml

```diff
@@ -104,8 +129,17 @@ jobs:
           fi
 
       - name: Run Dockle container linter
-        uses: hands-lab/dockle-action@v1
+        uses: erzz/dockle-action@v1.3.1
         with:
           image: ${{ steps.build-image.outputs.image }}
           exit-code: '1'
-          exit-level: WARN
+          failure-threshold: WARN
+          accept-filenames: ${{ env.DOCKLE_ACCEPT_FILES }}
+
+      - name: Save output to workflow summary
+        if: always() # Runs even if there is a failure
+        run: |
+          echo "# Dockle Results" >> $GITHUB_STEP_SUMMARY
+          if [ -f dockle-report.json ]; then
```

Comment:
same question here whether the if statement is needed

Reply:
I had it from the filtering so that it wouldn't print the header of the table with no rows, but since we aren't using it now, it isn't necessary